test/testETHAPI.js

```diff
@@ -275,4 +278,36 @@ describe('ETH API', function() {
     })
   })
 
+  describe('getCode', function() {
+    it('contract with empty code', function(done) {
+      eth.getCode({
+        rpcaddr,
+        rpcport,
+        address: addressWithoutContract,
+      }).then((result) => {
+        result.should.be.a.String()
+        if (result !== '0x' && result !== '0x0') {
```

Comment:
better to wrap this check in function and reuse it in both tests
example of function:
```
function checkZeroResult(value) {
  return ((value === '0x') || (value === '0x0'))
}
```

Reply:
still what is `0x` or `0x0` not clear